tests/Integration/GameSessionEntitiesTest.php

```diff
@@ -307,10 +304,24 @@ public function testPlan(): void
         $plan2 = $this->normalizer->denormalize($planFromConfig, Plan::class);
         $plan2->setPlanDescription('test description');
         $plan2->setCountry($this->em->getRepository(Country::class)->find($planFromConfig['plan_country_id']));
-        $planLayer = new PlanLayer();
-        $planLayer->setLayer(
-            (new Layer())->setOriginalLayer($this->em->getRepository(Layer::class)->find(1))
+        $plan2->setPlanState('APPROVED');
+        $derivedLayer = new Layer();
+        $derivedLayer->setOriginalLayer($this->em->getRepository(Layer::class)->find(1));
+        $geometry = new Geometry();
+        $geometry->setGeometryGeometry(
+            '[[1800176.69845479,748903.878],[4800176.69845479,2483199.127],[7398173.756,2483199.127]]'
```

Comment:
would it be possible to support [polygon1,polygon2] where a polygon is [point1,point2] and a point is [x,y]?
This is a valid format in geojson and wkt ,where the second onward polygons are holes
Don't think we support it at the moment so working around it